people.org

```diff
@@ -83,9 +85,9 @@ People to keep in loop:
 
 Drivers that will get XDP and use page_pool:
  - driver: mlx5
```

Comment:
Is mlx5 really still in 'will get XDP and use page_pool' state? (ping @netoptimizer )

Could we update this along with the rest?

Reply:
Any update? Can we update mlx5 in a different PR?
I got permission from Lorenzo to update his drivers, but never talked with anyone about mlx5.

Reply:
Sorry for late answer.
Sure we can update mlx5 in a different PR.

The file drivers.org seems very out of date :-(
Can you update it some more?